pkgs/development/python-modules/pytmx/default.nix

```diff
@@ -37,6 +41,7 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [
     pytestCheckHook
+    setuptools-scm
```

Comment:
should be in `nativeBuildInputs`

Reply:
I have adapted it. Thanks for your review and please excuse my very late reply. Somehow your feedback went through.